### DISCUSSION TITLE
useListNavigation keep list item as tabIndex 0

### DISCUSSION THREAD

**Discussion by dkilgore-eightfold** [floating-ui/floating-ui#2662] [Q&A] [answered]
useListNavigation toggles the z index of each element programmatically. I have a use case where I need them all to remain at 0. Is there a property I can use to achieve this? I need the isNested functionality, and it seems the only element that gets toggled to -1 is the submenu button in the list.

**Comment by atomiks** [comment]
`useListNavigation` doesn't set `tabIndex`, in the examples it's set manually. Only `Composite` sets it automatically.

**Reply by dkilgore-eightfold** [reply]
I'm using [this Cascading Menu example](https://codesandbox.io/s/bold-panna-9tf226?file=/src/DropdownMenu.tsx:5531-5623) as a boilerplate and it seems to be set automatically when navigating via arrow keys.

**Reply by dkilgore-eightfold** [reply]
This behavior is expected typically. However, my implementation calls for a consistent tab indices of 0.

**Reply by atomiks** [reply, marked as answer]
The `MenuItem` component sets it manually. So you can just remove the `tabIndex` prop or set it to `0` yourself.
